.github/workflows/test-ci.yml

```diff
@@ -34,7 +34,10 @@ jobs:
         pip install -e .
         pip install "gym[atari,accept-rom-license]"
         conda list
+    - name: Install test dependencies
+      run: |
+        pip install pytest
```

Comment:
I think we should add this to setup.py also since we will want our users to run tests too.
Or have a special dev dependecies section in setup.py (not sure how this works)

Reply:
setup.py will run when people run 'pip install sample-factory'?

Reply:
I actually added it to [dev] section of setup.py. Don't worry about it